services/src/handlers/wfs.rs

```diff
@@ -1180,14 +1180,14 @@ x;y
                     "vector": {
                         "type": "OgrSource",
                         "params": {
-                            "data": ne_10m_ports_id,
+                            "data": "points_with_time",
```

Comment:
is it a good idea to hardcode this her? What if we want to change it in the future? Then we'd have to change it everywhere.

Reply:
Do you have an idea on not to hardcode this? I could change it, then.